src/index.js

```diff
@@ -1,70 +1,699 @@
-const {
-  BaseKonnector,
-  saveFiles,
-  saveBills,
-  log
-} = require('cozy-konnector-libs')
-const lib = require('./lib')
-const parse = require('date-fns').parse
-
-module.exports = new BaseKonnector(start)
-
-// The start function is run by the BaseKonnector instance only when it got all the account
-// information (fields). When you run this connector yourself in "standalone" mode or "dev" mode,
-// the account information come from ./konnector-dev-config.json file
-async function start(fields) {
-  log('info', 'Authenticating ...')
-  await authenticate(fields.login, fields.password)
-  log('info', 'Successfully logged in')
-  const paymentNotices = await fetchPaymentNotices()
-  const insuranceCertificates = await fetchInsuranceCertificates()
-  const payments = await fetchPayments()
-  await saveBills(payments, fields, {
-    identifiers: ['MACIF'],
-    contentType: 'application/pdf'
-  })
-  await saveFiles(
-    [...paymentNotices, ...insuranceCertificates],
-    fields.folderPath,
-    {
-      contentType: 'application/pdf'
-    }
-  )
-}
+import { ContentScript } from 'cozy-clisk/dist/contentscript'
+import Minilog from '@cozy/minilog'
+import waitFor, { TimeoutError } from 'p-wait-for'
+import { parse } from 'date-fns'
+const log = Minilog('ContentScript')
+Minilog.enable('macifCCC')
 
-// this shows authentication using the [signin function](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_signin)
-// even if this in another domain here, but it works as an example
-async function authenticate(login, password) {
-  // setup cookies
-  log('info', 'Setup cookies...')
-  await lib.setupCookies()
+const baseUrl = 'https://www.macif.fr'
+// let FORCE_FETCH_ALL = false
 
-  log('info', 'Logging in...')
-  await lib.login(login, password)
-}
+const personnalInfos = []
+const personIdentity = []
+const attestationsInfos = []
+const schedulesInfos = []
 
-async function fetchPaymentNotices() {
-  const res = JSON.parse(await lib.fetchPaymentNotices())
-  return lib.parsePaymentNotices(res)
+var openProxied = window.XMLHttpRequest.prototype.open
```

Comment:
Now you should put interception code in its own file, like in the orange konnector

Reply:
Ok, on it's way 👌